src/components/ReachOut.js

```diff
@@ -1,10 +1,107 @@
-import styles from './reachout.module.css'
+import { useRef, useState } from 'react';
+import styles from './reachout.module.css';
+import emailjs from '@emailjs/browser';
+import send from '../icons/send.svg'
 
 function ReachOut() {
+  const form = useRef();
+  const [formValues, setFormValues] = useState({
+    contactName: '',
+    contactEmail: '',
+    messageSubject: '',
+    messageBody: ''
+  });
+
+  const handleForm = (e) => {
+    const {name, value} = e.target
+    setFormValues({
+      ...formValues,
+      [name]: value,
+    });
+  };
+
+  const sendEmail = (e) => {
+    e.preventDefault()
+    if (formValues.contactName && formValues.contactEmail && formValues.messageBody !== ''){
+      emailjs
+        .sendForm('service_9s7vueh', 'template_9kwp1h4', form.current, {
+          publicKey: '1FRD8OfceVnIcjUZK',
+        })
+        .then(
+          () => {
+            console.log('SUCCESS!');
+            setFormValues({
+              contactName: '',
+              contactEmail: '',
+              messageSubject: '',
+              messageBody: ''
+            });
+          },
+          (error) => {
+            console.log('FAILED...', error.text);
+          },
+        );
+      } else {
+        console.log("Name, email, and message cannot be blank")
+      }
```

Comment:
There's nothing "wrong" with this really, but I would suggest updating this to be a try/catch statement. They're the "newer" way of doing it, but mainly it's a lot easier to read and to avoid callback hell.


```suggestion
  const sendEmail = async (e) => {
    e.preventDefault()

    if (
      formValues.contactName
      && formValues.contactEmail
      && formValues.messageBody
    ) {
      try {
        await emailjs.sendForm(
          'service_9s7vueh',
          'template_9kwp1h4',
            form.current, {
            publicKey: '1FRD8OfceVnIcjUZK',
          }
        )

        setFormValues({
          contactName: '',
          contactEmail: '',
          messageSubject: '',
          messageBody: ''
        });
      } catch (error) {
        // eslint-disable-next-line no-console
        console.log('FAILED...', error.text);
      }
    } else {
      // eslint-disable-next-line no-console
      console.log("Name, email, and message cannot be blank")
    }
  };
```

Reply:
Spoke with John about this and we're going to work on it later.